shell.nix

```diff
@@ -24,21 +24,7 @@ let defaultCompiler = "ghc" + lib.replaceStrings ["."] [""] haskellPackages.ghc.
             then ourHaskell.packages.${defaultCompiler}
             else ourHaskell.packages.${compiler};
 
-    packages = p: [ p.haskell-language-server
-                    p.ghcide
-                    p.shake-bench
-                    p.hie-compat
-                    p.hls-plugin-api
-                    p.hls-brittany-plugin
-                    p.hls-class-plugin
-                    p.hls-haddock-comments-plugin
-                    p.hls-eval-plugin
-                    p.hls-explicit-imports-plugin
-                    p.hls-hlint-plugin
-                    p.hls-retrie-plugin
-                    p.hls-splice-plugin
-                    p.hls-tactics-plugin
-                  ];
+    packages = p: with builtins; map (name: p.${name}) (attrNames ourSources);
```

Comment:
Given that the list of packages is gone, I think the comment in lines 7-9 needs tweaking